src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -304,9 +304,7 @@ class KotlinPoetTest {
         |
         |import java.lang.String
         |
-        |fun String.shrink(): String {
-        |  return substring(0, length - 1)
-        |}
+        |fun String.shrink(): String = substring(0, length - 1)
```

Comment:
It's difficult to do well. We’d need to either infer type information (probably impossible) or have users supply hints (awkward).

We could also make it something you configure manually. I don’t love that option either.